pkg/controllers/jobflow/state/metrics.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2025 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package state
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"volcano.sh/volcano/pkg/scheduler/metrics"
```

Comment:
Should not import scheduler pkg, see https://github.com/volcano-sh/volcano/issues/4200

Reply:
Hmmm... Is there any other solution? Which package should we use? 🤔

Reply:
Here we just use the namespce constant in scheduler metrics pkg , we can refine one in controller.

Reply:
Thanks @Monokaix 
Got it
Fixed